server/router/server.go

```diff
@@ -305,6 +305,16 @@ func NewRouter(_ context.Context, h models.HandlerInterface, port int, g http.Ha
 	gMux.Handle("/api/integrations/credentials", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.DeleteUserCredential), models.ProviderAuth))).
 		Methods("DELETE")
 
+	// Handlers for User Connections
+	gMux.Handle("/api/integrations/connection", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.SaveConnection), models.ProviderAuth))).
+		Methods("POST")
+	gMux.Handle("/api/integrations/connections/{connectionKind}", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetConnections), models.ProviderAuth))).
+		Methods("GET")
+	gMux.Handle("/api/integrations/connection/{connectionKind}/edit", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.UpdateConnection), models.ProviderAuth))).
+		Methods("PUT")
+	gMux.Handle("/api/integrations/connection/{connectionKinId}/delete", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.DeleteConnection), models.ProviderAuth))).
```

Comment:
rename to `connectionId`

Reply:
@Azanul I've just checked it. It is `connectionId`

Reply:
@abdullah1308 Now I see what you mean. I thought you were talking about the PUT request. I'll update it right away.